Public/Src/Engine/UnitTests/Scheduler/PipExecutorTest.cs

```diff
@@ -1033,6 +1033,94 @@ public async Task FailingProcessWithChunkSizeErrorAndErrorRegex()
                 pathTable => GetConfiguration(pathTable, enableLazyOutputs: false, outputReportingMode: OutputReportingMode.FullOutputOnError));
         }
 
+        [Theory]
+        [InlineData(true, EventId.PipProcessError)]
+        [InlineData(false, EventId.PipProcessError)]
+        [InlineData(false, EventId.PipProcessError, 10 * SandboxedProcessPipExecutor.OutputChunkInLines)]
+        [InlineData(true, EventId.PipProcessWarning)]
+        [InlineData(false, EventId.PipProcessWarning)]
+        public async Task ProcessPrintPathsToLog(bool regexMatchesEverything, EventId eventId, int errorMessageLength = 0)
+        {
+            const string BadContents = "Anti-Matches!";
+
+            await WithCachingExecutionEnvironment(
+                GetFullPath(".cache"),
+                async env =>
+                { 
+                    string workingDirectory = GetFullPath("work");
+                    AbsolutePath workingDirectoryAbsolutePath = AbsolutePath.Create(env.Context.PathTable, workingDirectory);
+
+                    string destination = GetFullPath("dest");
+                    AbsolutePath destinationAbsolutePath = AbsolutePath.Create(env.Context.PathTable, destination);
+
+                    File.WriteAllText(destination, BadContents);

```

Comment:
>File.WriteAllText(destination, BadContents); [](start = 20, length = 44)

I'm not seeing why its necessary to write content to this file prior to launching the pip